utilities/console.py

```
@@ -56,16 +57,37 @@ def connect(self):
-                LOGGER.info(f"{self.vm.name}: Using password {self.password}")
-                self.child.sendline(self.password)
-
-        self.child.expect(self.prompt, timeout=150)
-        LOGGER.info(f"{self.vm.name}: Got prompt {self.prompt}")
+            attempts = 0
+            max_attempts = 5
+            prompts = self.prompt if isinstance(self.prompt, (list, tuple)) else [self.prompt]
+            prompts = list(prompts)  # Ensure it's always a list for concatenation
+            while attempts < max_attempts:
+                idx = self.child.expect(
+                    [self.login_prompt, "Password:"] + prompts + [pexpect.EOF, pexpect.TIMEOUT],
```

Ruff (0.14.1)

Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)